Makefile

```diff
@@ -160,6 +163,14 @@ cephcsi: check-env
 e2e.test: check-env
 	go test $(GO_TAGS) -mod=vendor -c ./e2e
 
+#
+# Update the generated deploy/scc.yaml when the template changed.
+# This requires running 'go mod vendor' so update the API files under the
+# vendor/ directory.
+deploy/scc.yaml: api/deploy/ocp/scc.yaml
```

Comment:
this should be some generic command (`generate-templates`) that will generate all artifacts?

Reply:
Yes, and probably it should be maintained in a Makefile under the `deploy/` directory as all yaml files need to be listed there.

Reply:
sounds good :+1: